hudi-spark-datasource/hudi-spark/src/main/scala/org/apache/spark/sql/hudi/command/procedures/RunClusteringProcedure.scala

```diff
@@ -105,35 +105,41 @@ class RunClusteringProcedure extends BaseProcedure
     logInfo(s"Pending clustering instants: ${pendingClustering.mkString(",")}")
 
     val client = HoodieCLIUtils.createHoodieClientFromPath(sparkSession, basePath, conf)
```

Comment:
var client = null out of try block and put `client = HoodieCLIUtils.createHoodieClientFromPath(sparkSession, basePath, conf)` into try block.

Reply:
done